src/wp-includes/functions.php

```diff
@@ -3322,10 +3334,13 @@ function _wp_json_prepare_data( $data ) {
  * @param int   $status_code The HTTP status code to output.
  */
 function wp_send_json( $response, $status_code = null ) {
-	@header( 'Content-Type: application/json; charset=' . get_option( 'blog_charset' ) );
+	if ( ! headers_sent() ) {
+		header( 'Content-Type: application/json; charset=' . get_option( 'blog_charset' ) );
 	if ( null !== $status_code ) {
```

Comment:
This if statement should be indented under the new if added by the PR.

Reply:
Indentation is incorrect, will update the PR.